### PR TITLE
Replace individual code owners with teams

### PR DESCRIPTION
**Pull request by pbochynski**
**Description**

Changes proposed in this pull request:

- propose new CODEOWNERS file structure based on area teams

**Related issue(s)**
- https://github.com/kyma-project/community/issues/613

### CODE REVIEW

File: CODEOWNERS [hunk 1]
```diff
@@ -279,16 +239,16 @@ components/etcd-tls-setup-job/ @PK85 @piotrmiskiewicz @ksputo @szwedm @voigt @wo
 /tests/end-to-end/upgrade/pkg/tests/rafter/ @m00g3n @pPrecel @dbadura @tgorgol @rJankowski93
 
 # All .md files
-*.md @klaudiagrz @mmitoraj @majakurcius @alexandra-simeonova @NHingerl
+*.md @kyma-project/documentation
```
Comment (mmitoraj) [inline CODEOWNERS:242]: The team's name is `technical-writers`
```suggestion
*.md @kyma-project/technical-writers
```

File: CODEOWNERS [hunk 2]
```diff
@@ -279,16 +239,16 @@ components/etcd-tls-setup-job/ @PK85 @piotrmiskiewicz @ksputo @szwedm @voigt @wo
...
 
 # All files and subdirectories in /docs
-/docs/ @klaudiagrz @mmitoraj @majakurcius @alexandra-simeonova @NHingerl
+/docs/ @kyma-project/documentation
```
Comment (mmitoraj) [inline CODEOWNERS:245]: The team's name is `technical-writers`
```suggestion
/docs/ @kyma-project/documentation
```

File: CODEOWNERS [hunk 3]
```diff
@@ -279,16 +239,16 @@ components/etcd-tls-setup-job/ @PK85 @piotrmiskiewicz @ksputo @szwedm @voigt @wo
...
 
 # Owners of the .kyma-project-io folder
 /.kyma-project-io/ @m00g3n @pPrecel @dbadura @tgorgol @rJankowski93
 
 # Config file for MILV - milv.config.yaml
-milv.config.yaml @m00g3n @pPrecel @klaudiagrz @mmitoraj @majakurcius @alexandra-simeonova @NHingerl
+milv.config.yaml @m00g3n @pPrecel @kyma-project/documentation
```
Comment (mmitoraj) [inline CODEOWNERS:251]: The team's name is `technical-writers`
```suggestion
milv.config.yaml @m00g3n @pPrecel @kyma-project/technical-writers
```

**Review by mmitoraj — CHANGES_REQUESTED** [review]

**Comment by pbochynski** [comment]
We go with slightly different approach: https://github.com/kyma-project/kyma/pull/13543
This work won't be continued.

**Pull request closed without merging**